activerecord/test/cases/encryption/encryption_schemes_test.rb

```diff
@@ -4,6 +4,13 @@
 require "models/author_encrypted"
 
 class ActiveRecord::Encryption::EncryptionSchemesTest < ActiveRecord::EncryptionTestCase
+  def run(*)
+    ActiveRecord::Encryption.config.add_to_filter_parameters = false
+    super
+  ensure
+    ActiveRecord::Encryption.config.add_to_filter_parameters = true
+  end
```

Comment:
We probably want this to be false by default, and turned on when the application boots.